homeassistant/components/shelly/device_trigger.py

```diff
@@ -108,9 +111,9 @@ async def async_validate_trigger_config(
 
 async def async_get_triggers(
     hass: HomeAssistant, device_id: str
-) -> list[dict[str, str]]:
+) -> GetAutomationsResult:
```

Comment:
I would like to revert this, for reasoning and the review see: <https://github.com/home-assistant/core/pull/72163#issuecomment-1133865649>

Reply:
PR to revert: https://github.com/home-assistant/core/pull/72303